recipe/meta.yaml

```diff
@@ -1,50 +1,66 @@
 {% set name = "google-cloud-resource-manager" %}
-{% set version = "1.8.1" %}
+{% set version = "1.12.5" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/google-cloud-resource-manager-{{ version }}.tar.gz
-  sha256: 9e94eff9fe77dc92bf27671e27a35aee64bdd077ca6873919869c5cf52c1ccb4
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name.replace('-', '_') }}-{{ version }}.tar.gz
+  sha256: b7af4254401ed4efa3aba3a929cb3ddb803fa6baf91a78485e45583597de5891
 
 build:
   number: 0
-  noarch: python
-  script: {{ PYTHON }} -m pip install . -vv
+  skip: true  # [py<37]
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
 
 requirements:
   host:
-    - python >=3.7
+    - python
     - pip
+    - setuptools
+    - wheel
   run:
-    - python >=3.7
-    - google-api-core-grpc >=1.34.0,<3.0.0dev,!=2.0.*,!=2.1.*,!=2.2.*,!=2.3.*,!=2.4.*,!=2.5.*,!=2.6.*,!=2.7.*,!=2.8.*,!=2.9.*,!=2.10.*
-    - proto-plus >=1.22.0,<2.0.0dev
-    - protobuf >=3.19.5,<5.0.0dev,!=3.20.0,!=3.20.1,!=4.21.0,!=4.21.1,!=4.21.2,!=4.21.3,!=4.21.4,!=4.21.5
+    - python
+    - google-api-core >=1.34.1,<3.0.0dev,!=2.0.*,!=2.1.*,!=2.2.*,!=2.3.*,!=2.4.*,!=2.5.*,!=2.6.*,!=2.7.*,!=2.8.*,!=2.9.*,!=2.10.*
+    - google-api-core-grpc
+    # Exclude incompatible versions of `google-auth`
+    # See https://github.com/googleapis/google-cloud-python/issues/12364
+    - google-auth >=2.14.1,<3.0.0dev,!=2.24.0,!=2.25.0
+    - proto-plus >=1.22.3,<2.0.0dev
+    - protobuf >=3.20.2,<6.0.0dev,!=4.21.0,!=4.21.1,!=4.21.2,!=4.21.3,!=4.21.4,!=4.21.5
     - grpc-google-iam-v1 >=0.12.4,<1.0.0dev
 
 test:
   imports:
     - grpc
     - google.cloud.resourcemanager
     - google.cloud.resourcemanager_v3
+  source_files:
+    - tests
   commands:
     - pip check
+    - set PROTOCOL_BUFFERS_PYTHON_IMPLEMENTATION=python     # [win]
+    - export PROTOCOL_BUFFERS_PYTHON_IMPLEMENTATION=python  # [not win]
```

Comment:
Are these well-known settings we have to apply?

Reply:
I guess, yes. There are three possible variants - python, cpp, and ubp https://github.com/googleapis/google-cloud-python/blob/5e3f4aebeb2f79efb1992ae623eb1aea86de2b0c/packages/grafeas/noxfile.py#L165-L166

Reply:
We should state why these variables are required and, given there's a choice of values, why we have chosen a particular one.
So that the next maintainer isn't scratching their head.

Reply:
```suggestion
    # noxfile.py#L199
    - set PROTOCOL_BUFFERS_PYTHON_IMPLEMENTATION=python     # [win]
    - export PROTOCOL_BUFFERS_PYTHON_IMPLEMENTATION=python  # [not win]
```